files/zh-cn/.markdownlint-cli2.jsonc

```
@@ -0,0 +1,73 @@
+{
+  "config": {
+    "extends": "../../.markdownlint-cli2.jsonc",
+    "search-replace": {
+      "rules": [
+        {
+          "name": "nbsp",
+          "message": "Don't use no-break spaces",
+          "searchPattern": "/ /g",
+          "replace": " ",
+          "searchScope": "all"
+        },
+        // XXX Many instances still found in translated content
+        // {
+        //   "name": "m-dash",
+        //   "message": "Don't use '--'. Use m-dash — instead",
+        //   "search": " -- ",
+        //   "replace": "——",
+        //   "searchScope": "text"
+        // },
+        // {
+        //   "name": "relative-link",
+        //   "message": "Internal links should start with '/'",
+        //   "searchPattern": "/([\\(\"])(en-US|zh-cn)\\/docs/gi",
```

Although the regex ignores case, maybe using the "correct" case is better. This is just a suggestion :)

```suggestion
        //   "searchPattern": "/([\\(\"])(en-US|zh-CN)\\/docs/gi",
```